lib/pbio/test/src/test_animation.c

```diff
@@ -15,45 +15,49 @@
 
 #define TEST_ANIMATION_TIME 10
 
-PROCESS_NAME(pbio_light_animation_process);
-
 static uint8_t test_animation_set_hsv_call_count;
 
 static uint32_t test_animation_next(pbio_light_animation_t *animation) {
     test_animation_set_hsv_call_count++;
     return TEST_ANIMATION_TIME;
 }
 
-static PT_THREAD(test_light_animation(struct pt *pt)) {
-    PT_BEGIN(pt);
+#define YIELD(state)      \
```

Comment:
We've only needed it in this test so far, but we should probably move this to the `pbio/os` module.